realsense2_camera/test/live_camera/test_camera_all_profile_tests.py

```diff
@@ -0,0 +1,235 @@
+# Copyright 2023 Intel Corporation. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+import sys
+import itertools
+
+
+import pytest
+import rclpy
+
+from sensor_msgs.msg import Image as msg_Image
+from sensor_msgs.msg import Imu as msg_Imu
+from sensor_msgs.msg import PointCloud2 as msg_PointCloud2
+
+import numpy as np
+
+sys.path.append(os.path.abspath(os.path.dirname(__file__)+"/../utils"))
+import pytest_rs_utils
+from pytest_rs_utils import launch_descr_with_parameters
+from pytest_rs_utils import delayed_launch_descr_with_parameters
+from pytest_rs_utils import get_rosbag_file_path
+import pytest_live_camera_utils
+from rclpy.parameter import Parameter
+from rcl_interfaces.msg import ParameterValue
+from rcl_interfaces.srv import SetParameters, GetParameters, ListParameters
+from pytest_live_camera_utils import debug_print
+def check_if_skip_test(profile, format):
+    '''
+    if profile == 'Color':
+        if "BGRA8" == format:
+            return True
+        if "RGBA8" == format:
+            return True
+        if "Y8" == format:
+            return True
+    elif profile == 'Depth':
+        if "Z16" == format:
+            return True
+    elif profile == 'Infrared':
+        if "Y8" == format:
+            return True
+        if "Y16" == format:
+            return True
+        if "BGRA8" == format:
+            return True
+        if "RGBA8" == format:
+            return True
+        if "Y10BPACK" == format:
```

Comment:
on which camera model do you see this format ?

Reply:
in D415:     
- Infrared 1:	  1920x1080	@ 30Hz	   Y10BPACK